pages/birthdays/index.js

```diff
@@ -17,7 +17,7 @@ const StyledDiv = styled.div`
   align-items: center;
 `;
 
-const StyledBirthdayList = styled.div`
+const StyledBirthdayList = styled.article`
```

Comment:
I am not sure why you have changed so many elements to `article` elements. I see that you try to adopt @JessicaLoers suggestion to use semantic HTML, but is the `article` element to solution in all these cases?

Reply:
hi Paul. Yes I have looked at many possibilities today and also talked again with coach mareike and stay for now here at br... but in the future I now know much more possibilities :)

You're right, I was a bit hectic with all the articles. I have now used sections and article... Thanks